ui/main/src/app/modules/core/application-loading/application-loading.component.ts

```diff
@@ -176,13 +176,21 @@ export class ApplicationLoadingComponent implements OnInit {
         this.store.select(selectIdentifier).subscribe((identifier) => {
             if (identifier) {
                 this.logger.info(`User ${identifier} logged`);
+                this.synchronizeUserTokenWithOpfabUserDatabase();
                 this.showLoginScreen = false;
                 this.userLogin = identifier;
                 this.loadSettings();
             }
         });
     }
 
+    private synchronizeUserTokenWithOpfabUserDatabase() {
+        this.userService.synchronizeWithToken().subscribe({
+            next: () =>  this.logger.info("Synchronization of user token with user database done"),
+            error: () => this.logger.warn("Impossible to synchronize user token with user database")
```

Comment:
Is it correct to just log a warn ? Previously in case of error in synchronization the user would have been logged out by calling "authService.clearAuthenticationInformation()"

Reply:
yes that was the previous behavior . But i prefer continuing the loading process if not succeed as opfab can work without the synchro